xAPI.md

```diff
@@ -507,6 +507,41 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
+This section explains and shows graphically how different pieces of xAPI can fit together.  
+
+##### Tracking Experiences Through Systems
+
+![Data Flow in xAPI](/xAPIDataFlow.jpg)
+
+Figure 1 shows the tracking of experiences through the data flow of xAPI. While, a learner experiences a learning activity, data about that experience is tracked to a Learning Record Store (LRS).  Authentication with the LRS may use the learner's credentials, the activity provider's credentials, or a combination.  A Data Analysis Tool, acting as a Service with permission, can request the data from the LRS and use it to display meaningful reports or visualizations. In the case of more structured learning, a user may be dependent on an Activity Provider to deliver and/or authorize use of content to them. Once this triangle of authorization and access is complete, a registration exists for that user in that content tracked by that LRS and experience tracking may proceed.
+
+##### Activity Data and Metadata
+
+![xAPI Activity Data and Metadata](/xAPIMetadata.jpg)
+
+Understanding how a single activity (as uniquely identified by its IRI) is produced, populated, updated, and 
+consumed is a key concept in xAPI. Figure 2 shows this process.  xAPI Statements with the Activity objects have  
+metadata properties that may be populated.  This is done through the  Activity Definition Object.  Within the 
+Activity is an IRI which also could have metadata located at where it resolves to. (Note that this is different  
+from the moreinfo  URL in the Activity Definition, which could also have information located at it, but is not 
+involved in any way in this information flow.) The base portion of the IRI as well as any metadata where the IRI resolves is under the control of the Metadata Provider.  Any metadata at the IRI is the authoritative source of metadata, and could be used to populate the LRS's aggregated version of the Activity's metadata (Activity Definition) instead of what it receives from Statements. A Metadata Consumer can access Metadata directly from the authority or can query the Activities Resource.  
```

Comment:
I think we can delete "base portion of the" - consider registry or purl IRIs, for example where the base is not under the control of the person registering the identifier. 

Change "aggregated" to "canonical".

"instead of" is technically too strong because it might compliment what it gets in statements in the case of multi-language definitions. "with priority over"? We use the phrase "strong preference" elsewhere I believe, though I'm not sure that fits here. "Alongside"? "On top of"? "In place of"? - I'm not really happy with any of these.